daemon/api.go

```diff
@@ -67,81 +67,80 @@ var (
 	rootCmd = &Command{
 		Path:    "/",
 		GuestOK: true,
-		GET:     SyncResponse([]string{"/1.0"}).Self,
+		GET:     SyncResponse([]string{"TBD"}).Self,
```

Comment:
it's supposed to be a human-readable page, but isn't (yet). Follow-up-branch work.